principles.md

```diff
@@ -0,0 +1,20 @@
+## Product and Engineering Principles
+
+**We believe in being paid for quality work**
+
+There's an incredible amount of software infrastructure that is under funded and ignored.
+We believe our software needs to generate revenue to employ the best minds.
+
+**We balance openness and transparency**
+
+We work as much as we can in the open; our commits, designs, and comments are largely in public repositories for anyone to use and observe.
+We strive to find balance in open-core and community friendly software versus competing in a dynamic and evolving market segment.
+
+All our security critical code is developed in the open.
+Our independent audits [get published](https://goteleport.com/resources/audits/), even the unflattering ones.
+
+**We want to tackle and solve the hard problems**
+
+We want to minimize the theater in security products, and offer meaningful improvements to security posture.
+We want to focus on the user, having a product that users aren't trying to work around to get their jobs done.
```

Comment:
```suggestion
We want to optimize for user experience, and deliver a product that stays out of our users' way so that they can get their jobs done.
```